data/mods/Aftershock/player/professions.json

```diff
@@ -434,5 +434,31 @@
     "traits": [ "PROF_MED", "UPLIFTED", "THRESH_CEPHALOPOD", "CEPH_EYES", "GILLS", "INT_UP_2", "LEG_TENTACLES", "SLIMY" ],
     "//": "Need to add underwater gear for this.  See extended leg_tentacles mutation.",
     "items": { "both": { "items": [ "wetsuit_cecalia", "trident", "dive_bag" ] } }
+  },
+  {
+    "type": "profession",
+    "id": "robot_scavenger",
+    "name": "Robot Scavenger",
+    "description": "Before the cataclysm struck, you were one of the members of an underground group striving to free cyborgs from their shackles, raiding laboratories and underground clinics in your pursuit of liberation.  Now that most of the humanity has been turned into zombies, your goal has become even more important to you.",
+    "points": 4,
+    "skills": [
+      { "level": 2, "name": "gun" },
+      { "level": 2, "name": "rifle" },
+      { "level": 3, "name": "computer" },
+      { "level": 3, "name": "electronics" }
+    ],
+    "items": {
+      "both": {
+        "items": [ "jumpsuit", "hazmat_suit", "mask_gas", "socks", "molle_pack", "wristwatch", "bot_manhack", "bot_manhack" ],
```

Comment:
Oh, curious here as well, why hazmat suit?

Reply:
Hazmat suit is probably a good stand in until the exoplanet is standard and we need to equip people with magellan suits at the beginning of the game?

Reply:
Mostly a leftover from original profession idea with lab raider vibe, but probably still works as this profession would have to frequently deal with skitter bots